quarkchain/cluster/root_state.py

```diff
@@ -281,6 +281,7 @@ def get_tip_block(self):
         return self.db.get_root_block_by_hash(self.tip.get_hash())
 
     def add_validated_minor_block_hash(self, hash: bytes, coinbase_tokens: Dict):
+        Logger.info("Minor block received: {}".format(hash.hex()))
```

Comment:
instead of logging inside root state, I think we should log at the grpc response handling place, because we don't want to affect normal non-grpc setup